connectors/agentic-ai/src/test/java/io/camunda/connector/agenticai/aiagent/framework/langchain4j/ChatModelFactoryTest.java

```diff
@@ -157,6 +165,86 @@ static Stream<AnthropicModelParameters> nullModelParameters() {
     }
   }
 
+  @Nested
+  class AzureOpenAiChatModelFactoryTest {
```

Comment:
🔧 As we don't test this model configuration in e2e tests, could we add additional tests for completeness and consistency with the other models?

- no model parameters being set
- no authority host being set